mm2src/coins/lp_coins.rs

```diff
@@ -1395,6 +1430,96 @@ impl WithdrawError {
     }
 }
 
+#[derive(Serialize, Display, Debug, SerializeErrorType)]
+#[serde(tag = "error_type", content = "error_data")]
+pub enum SignatureError {
+    #[display(fmt = "Invalid request: {}", _0)]
+    InvalidRequest(String),
+    #[display(fmt = "Internal error: {}", _0)]
+    InternalError(String),
+    #[display(fmt = "Coin is not found: {}", _0)]
+    CoinIsNotFound(String),
+    #[display(fmt = "Message prefix not found")]
```

Comment:
Could you please change it to something like `sign_message_prefix is not set in coin config`?